py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
Does that make sense or you're saying I can python2-test things modules from the terminal? For instance, I'm not sure what I need to do with this. No issue w/ python3.
```
lasers~/src/py3status/py3status/modules git:(master) python2 nvidia_temp.py
Traceback (most recent call last):
  File "nvidia_temp.py", line 79, in <module>
    module_test(Py3status)
  File "/home/chris/src/py3status/py3status/module_test.py", line 52, in module_test
    print(getattr(module, method)(*method_args))
  File "nvidia_temp.py", line 48, in nvidia_temp
    format_units=self.format_units
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb0' in position 0: ordinal not in range(128)

lasers~/src/py3status/py3status/modules git:(master) python3 nvidia_temp.py
{'cached_until': 1485532095.0, 'color': '#00FF00', 'full_text': 'GPU: 61°C'}
{'cached_until': 1485532096.0, 'color': '#00FF00', 'full_text': 'GPU: 61°C'}
```